### ISSUE TITLE
Addon only partly installs

### ISSUE THREAD

**Issue by makkie2002** [hassio-addons/addon-node-red#826]
When installing the nodered addon as explained in the installation guide it seems that the installation went ok as the webinterface of node-red can be reached. Also editing and running of "flows" just goes fine. Even part of the integration with home-assistant is ok (eg. events get through to node-red). Only creating new entities from node-red to be available in home-assistant does not work. The first clue of an issue you get already when looking at the properties of the entity node where a warning is given:
`Attention: This node requires Node-RED custom integration to be installed in Home Assistant for it to function.`

When installing in HA also node-red can not be found in "integrations". Furthermore no nodered directory can be found in the "custom_components" directory. On an older HA installation I had no issues at all, so I decided to copy the nodered folder in custom_components from the old installation to the new installation, but this did not work either. Then I edited the core.config_entries file to include also a nodered entry (c/p from old installation) and then all started to work as it should. 

So I have the idea that something went wrong with the installation of the addon. I tried on a new installation (in a virtual machine) and also here the same issue appeared. So probably there is a bug in the installation process?

Nodered addon version 7.2.11
Home Assistant 2020.12.1
Home Assistant OS 5.8

![Aantekening 2020-12-25 001406](https://user-images.githubusercontent.com/5372859/103110614-b6e4c380-4647-11eb-8bd9-7ff2ce8b0d00.jpg)

**Comment by bonanitech** [comment]
https://github.com/zachowj/hass-node-red/blob/master/README.md#installation

**Comment by makkie2002** [comment]
Thanks bonanitech! That helped :). I remember trying to do this before but did not succeed to get it to show into "integrations". But after some more research and fiddling I found out that I needed to "clear the browser-cache". Did not know how to do that so I decided to switch to the Edge-browser and voilá there was the integration. Thanks for helping out!

Maybe a good idea to have this "Companion component" installed with the add-on as I think it is an essential component for integration of Node-red and Home Assisstant ...

Cheers & happy X-mas !
